sat/cli/bootprep/input/session_template.py

```diff
@@ -186,6 +186,12 @@ def get_create_item_data(self):
             boot_set_api_data.update(boot_set_data)
             api_data['boot_sets'][boot_set_name] = boot_set_api_data
 
+            # Fetch full boot sets
+            boot_sets = self.boot_set
+            # Render the rootfs_provider_passthrough using Jinja2
+            rendered_rootfs = self.jinja_env.from_string(
+                        boot_sets[boot_set_name]['rootfs_provider_passthrough']).render(self.data)
```

Comment:
Why do we pass in `self.data` to this? From what I can tell from the Jira, it is not required to allow it to render fields with arbitrary data from the session template. I think there's a way we can just modify and use the `jinja_rendered` decorator instead.

I think passing in `self.data` as the context when rendering is going to lead to some weird behavior and possibly hard-to-fix bugs, and then it will be hard to remove the ability to use the data if people start using it that way. I would suggest we not add this unless it's explicitly requested.